docs/GettingStartedDocs/GettingStarted.Windows.md

```diff
@@ -24,6 +18,14 @@ Windows.
 - Windows 10 64-bit with Fall Creators Update (1709)
```

Comment:
Nonblocking: I need to take an action item to test this on WS2016 since that is the configuration ACC deploys, and IIRC it cannot use the user-mode driver to pull down the required PSW, so that's a separate installation step.